python/cudf/dataframe/string.py

```diff
@@ -690,6 +690,18 @@ def default_na_value(self):
     def take(self, indices):
         return self.element_indexing(indices)
 
+    def binary_operator(self, binop, rhs, reflect=False):
+        lhs = self
+        if reflect:
+            lhs, rhs = rhs, lhs
+        if isinstance(rhs, StringColumn) and binop == 'add':
+            return lhs.data.cat(
+                others=rhs.data,
+            )
```

Comment:
You should retain the index of the `lhs` here.

Reply:
Nevermind, this is at the column level instead of the Series level. I'm losing it.